src/database/schema/ChatSchema.ts

```diff
@@ -313,19 +352,27 @@ class ChatSchema implements BaseDbSchema {
       user: null,
       userId,
       isMe: true,
-      isContinuous: true
+      isContinuous: true,
+      attachmentJson: JSON.stringify(attachments)
     });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` property is assigned without any validation or transformation. It would be safer to ensure that `attachmentJson` is always stored as a stringified JSON object.
```diff
-      isContinuous: true,
-      attachmentJson: JSON.stringify(attachments)
+      isContinuous: true,
+      attachmentJson: JSON.stringify(attachments || [])
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
No need, attachments default is []